app/models/pin.rb

```diff
@@ -0,0 +1,10 @@
+class Pin < ApplicationRecord
+  validates :description, presence: true
+  validates :image_url, presence: true
+  validates :article_url, presence: true
+  validates :board, presence: true
+
```

Comment:
So  if we set this up like Pinterest does, pins will no longer `belongs_to :board`

Instead they will `has_many :boards through :board_pins` ?